tools/uart_hw_test/README.md

```diff
@@ -40,8 +40,21 @@ It is MANDATORY to first run the uart_hw_test tash command and enter the port nu
 After this run the ./uart_test.sh test app on linux and enter the port number.
 If this sequence is not followed, then the result is not proper.
 
+### How to USE for windows
+1. Find the UART port to be tested. For example, on the RTK8721 board, we will perform uart test for the uart1.
+2. Connect the uart (Rx, Tx, Gnd) lines from the test device to the Windows PC using Serial-USB converter.
+3. Enable CONFIG_UART_HW_TEST config. Build and download the binary.
+4. After boot up, run the uart_hw_test command at the tash prompt (find the appropiate console for windows to get tash prompt). In our example, we use putty console and enter appropriate serial COM number and open the tash prompt .
+5. Input the appropriate port number . In our example, the uart1 appears as /dev/ttys2 on the device . On the PC, the Serial-USB converter appears as /dev/ttyUSB2 . Hence the port number is 2 in both cases.
+6. Now compile the uart_test_windows.c file in windows using "gcc -o executable_file uart_test_windows.c" and it will create executable_file.exe executable file.
```

Comment:
1. Where should we execute `gcc -o ...`? in the cmd?
2. To run `gcc`, do we need any prerequisite?
3. How about providing pre-built executables? I think there is no reason that user should build on time. (same for linux)

Reply:
1 . yes in command prompt in that directory 
2. No , gcc should be there in the system 
3. can do  , i will add the executable file also and also guidance to how build that

Reply:
> 3. can do , i will add the executable file also and also guidance to how build that

Could you do the same in Linux?

Reply:
for linux , there is already a "uart_test.sh" file which can be run on terminal and it will automatically create the executable file and run . So we dont need to do any extra thing.

Reply:
It requires `gcc`. To use gcc, do we need any prerequisite?

Reply:
if gcc is there in linux system , then its fine , we dont need any prerequisite , it will run by running ./uart_test.sh

Reply:
> if gcc is there in linux system , then its fine

Yes, `IF` gcc is there in linux. All of Linux has the gcc, then no need to add. But if any Linux does not have it, then it's the prerequisite.

Reply:
okay , I have added instruction in README file in step no 6 in steps for how to use in linux. I added that you need to install gcc if you dont have one before running ./uart_test.sh file